masterlist.yaml

```
@@ -4239,7 +4239,17 @@ plugins:
       - 'Enhanced Vegetation [110%].esp'
       - 'Enhanced Vegetation [125%].esp'
       - 'Enhanced Vegetation [150%].esp'
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'For this to work as intended, it''s necessary to raise the grass limit per texture (iMaxGrassTypesPerTexure) in the Oblivion ini - 5 should be more than enough.'
```

This message can be edited to use an existing, translated message anchor while still communicating enough of what needs to be communicated.

```suggestion
      - <<: *requiresX
        type: say
        subs: [ 'INI setting `iMaxGrassTypesPerTexure=5`' ]
```